live-examples/html-examples/forms/css/progress.css

```diff
@@ -0,0 +1,16 @@
+.progress-example {
+    display: table;
+}
+
+.progress-example div {
+    display: table-row;
+}
+
+label, progress {
```

Comment:
Formatting nit "When using multiple selectors in a rule declarations, give each selector its own line": https://mdn.github.io/mdn-fiori/patterns/css/formatting/#general